lab-gurpreet/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+var testFiles = ['test/**/*.js'];
+var appFiles = ['lib/**/*.js', 'bin/cowsay'];
+gulp.task('lint:app', () => {
+  gulp.src(appFiles)
+        .pipe(eslint({
+          rules: {
+            'indent': [2, 2]
+          },
+          envs: [
+            'node',
+            'es6'
+          ]
+        }))
+        .pipe(eslint.format());
```

Comment:
If you wish to include the .eslintrc options in your gulpfile, please make sure you're including all of the rules from the class .eslintrc, not just a subset.

Also, please make sure that you're including the class .eslintrc file in your project root. Travis will need it to run properly.